docs/docs/concepts/policies.mdx

```diff
@@ -2,5 +2,699 @@
+
+
+**What about TED?**
+
+There is no reason why you can't also have TED in your configuration. However,
+
+- TED frequently makes predictions with very high confidence values (~0.99) so
+  will often override what the `IntentlessPolicy` is doing.
+- TED and the `IntentlessPolicy` are trying to solve similar problems, so your
+  system is easier to reason about if you just use one or the other.
+
+## Steering the Intentless Policy
+
+The first step to steering the intentless policy is adding and editing responses
+in the domain file. Any response in the domain file can be chosen as an response
+by the intentless policy. This whitelisting ensures that your assistant can
+never utter any inappropriate responses.
+
+```yaml-rasa title="domain.yml"
+utter_faq_4:
+  - text:
+      We currently offer 24 currencies, including USD, EUR, GBP, JPY, CAD, AUD,
+      and more!
+utter_faq_5:
+  - text:
+      Absolutely! We offer a feature that allows you to set up automatic
+      transfers to your account while you're away. Would you like to learn more
+      about this feature?
+utter_faq_6:
+  - text:
+      You can contact our customer service team to have your PIN unblocked. You
+      can reach them by calling our toll-free number at 1-800-555-1234.
+```
+
+Beyond having the `utter_` prefix, the naming of the utterances is not relevant.
+
+The second step is to add
+[end-to-end stories](../building-classic-assistants/training-data-format.mdx#end-to-end-training)
+to `data/e2e_stories.yml`. These stories teach the LLM about your domain, so it
+can figure out when to say what.
+
+```yaml title="data/e2e_stories.yml"
+- story: currencies
+  steps:
+    - user: How many different currencies can I hold money in?
+    - action: utter_faq_4
+
+- story: automatic transfers travel
+  steps:
+    - user: Can I add money automatically to my account while traveling?
+    - action: utter_faq_5
+
+- story: user gives a reason why they can't visit the branch
+  steps:
+    - user: I'd like to add my wife to my credit card
+    - action: utter_faq_10
+    - user: I've got a broken leg
+    - action: utter_faq_11
+```
+
+The stories and utterances in combination are used to steer the LLM. The
+difference here to the existing policies is, that you don't need to add a lot of
+intent examples to get this system going.
+
+## Testing
+
+The policy is a usual Rasa Policy and can be tested in the same way as any other
+policy.
+
+### Testing interactively
+
+Once trained, you can test your assistant interactively by running the following
+command:
+
+```bash
+rasa shell
+```
+
+If a flow you'd like to implement doesn't already work out of the box, you can
+add try to change the examples for the intentless policy. Don't forget that you
+can also add and edit the traditional Rasa primitives like intents, entities,
+slots, rules, etc. as you normally would. The `IntentlessPolicy` will kick in
+only when the traditional primitives have low confidence.
+
+### End-to-End stories
```

Comment:
this section can be removed, no?

Reply:
on using e2e stories with the intentless policy? you still can, not sure I'd want to remove it

Reply:
ah ok. I got confused with end-to-end testing. I think that also what will be confusing is that end-to-end stories (and stories) are a thing of the past